src/components/ui/icon-button/icon-button.js

```diff
@@ -24,13 +24,38 @@ const IconButton = ({
   visible,
 }) => {
   const Icon = icon;
+  let timeout;
+  const [isTooltipVisible, setIsTooltipVisible] = useState(false);
 
   const labelPosition = labelPositionTypes.includes(
     labelTextPosition.toLowerCase()
   )
     ? labelTextPosition.toLocaleLowerCase()
     : 'right';
 
+  useEffect(() => {
+    window.localStorage.removeItem('kedro-viz-tooltip-show');
+  }, []);
+
+  const showTooltip = () => {
+    if (localStorage.getItem('kedro-viz-tooltip-show') === null) {
+      window.localStorage.setItem('kedro-viz-tooltip-show', true);
+      timeout = setTimeout(() => {
+        setIsTooltipVisible(true);
+      }, 1000);
+    } else {
+      setIsTooltipVisible(true);
+    }
+  };
+
+  const hideTooltip = () => {
+    setTimeout(() => {
```

Comment:
hey @cvetankanechevska   I think it would be nice to have a comment here to explain why the local storage needs to be reset this way, so whoever works on this component next will have some context.